src/client/ContentAccess.js

```diff
@@ -3086,13 +3086,11 @@ exports.VerifyContentObject = async function({libraryId, objectId, versionHash})
  * @param {string=} objectId - ID of the object
  * @param {string=} versionHash - Version hash of the object -- if not specified, latest version is returned
  * @param {string=} salt - base64-encoded byte sequence for salting the audit hash
- * @param {Array<number>=} samples - list of percentages used for sampling the content part list; up to 3 values max, 0.0-1.0 values
```

Comment:
Just FYI, the qfab API only allows up to 3 samples; if you specify more, it will fail

Reply:
got it, I will note it in the comment and silently truncate the list if it's too long